app/assets/stylesheets/templates/_step.scss

```diff
@@ -6,4 +6,15 @@
     content: "";
     display: block;
   }
+
+  .form-card {
+    max-width: 72rem;
+
+    @include media($tablet-up) {
+      margin: {
+        left: auto;
+        right: auto;
+      }
+    }
+  }
```

Comment:
I feel you on that, but they have only one element that that rule applies to. Since we'll have at least 2 I think it's a worthwhile endeavor :)